maintainers/maintainer-list.nix

```diff
@@ -19554,6 +19554,12 @@
     githubId = 7677567;
     name = "Victor SENE";
   };
+  vsoch = {
+    email = "contact@vsoch.com";
+    github = "vsoch";
+    githubId = 814322;
+    name = "Vanessa Sochat";
+  };
```

Comment:
This should be it's own commit called `maintainers: add vsoch`

Reply:
Done.